core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -366,6 +366,12 @@ class SparkContextSuite extends SparkFunSuite with LocalSparkContext with Eventu
     }
   }
 
+  test("add jar local path with comma") {
+    sc = new SparkContext(new SparkConf().setAppName("test").setMaster("local"))
+    sc.addJar("file://Test,UDTF.jar")
+    assert(!sc.listJars().exists(_.contains("UDTF.jar")))
```

Comment:
What does this test mean? Why do you do ` assert(sc.listJars().exists(_.contains("Test,UDTF.jar")))`?

Reply:
> What does this test mean? Why didn't you write it like ` assert(sc.listJars().exists(_.contains("Test,UDTF.jar")))`?

Mess in brain...